### ISSUE TITLE
NimbusML Archived?

### ISSUE THREAD

**Issue by jpepg4** [dotnet/machinelearning#5607]
I noticed that NimbusML has been archived.  As ML.NET evolves will Microsoft be providing a different path to run the platform through Python?  A couple of the models in ML.NET are unique and produce very good results but having this accessibility on the more commonly used Python platform is important for adaptation.

**Comment by mitar** [comment]
I agree, there does not seem to be any documentation/explanation about that repository being archived. [Documentation is still up](https://docs.microsoft.com/en-us/nimbusml/overview?view=nimbusml-py-latest). What is happening?

**Comment by saint4eva** [comment]
> I noticed that NimbusML has been archived. As ML.NET evolves will Microsoft be providing a different path to run the platform through Python? A couple of the models in ML.NET are unique and produce very good results but having this accessibility on the more commonly used Python platform is important for adaptation.

Or someone from python community can produce an api wrapper?
